src/foam/nanos/dig/DIG.js

```diff
@@ -31,68 +36,80 @@ foam.CLASS({
     }
   ],
 
+  imports: [
+    'nSpecDAO'
+  ],
+
   properties: [
-    'id',
     {
-      class: 'String',
+      name: 'id',
```

Comment:
I think we should add class: 'String'. How was this working on the java side without this?

Reply:
ID is already defined in the parent class .. DefaultHttpParameters.